JPF/mysql.skeleton.ipynb

```diff
@@ -108,6 +107,11 @@
     "pw = os.environ['MYSQL_PASSWORD']\n",
     "db = os.environ['MYSQL_DATABASE']\n",
     "\n",
+    "user = 'root'\n",
+    "host = 'localhost'\n",
+    "pw = ''\n",
+    "db = 'heatseek'\n",
+    "\n",
```

Comment:
Got it. No worries. It's not bad form to make changes like that while working locally, but just don't forget to pull them out when submitting code. Otherwise the next person who pulls will be quite confused as to why their config isn't working.